src/js/components/Export/ExportPoll.vue

```diff
@@ -143,7 +143,7 @@ export default {
 					} else if (style === 'raw') {
 						votesLine.push(this.getVote({ userId: participant.userId, option }).voteAnswer)
 					} else {
-						votesLine.push(this.getVote({ userId: participant.userId, option }).voteAnswerTranslated ?? t('polls', 'no'))

```

Comment:
In this case the translations in src/js/store/module/vote.js have to be adjusted too.

Reply:
https://github.com/nextcloud/polls/blob/1e4567b9cc6fa4eb3b8d7b256ef08edd6de8aadb/src/js/store/modules/votes.js#L97-L112

Reply:
This could be removed BTW
https://github.com/nextcloud/polls/blob/1e4567b9cc6fa4eb3b8d7b256ef08edd6de8aadb/src/js/store/modules/votes.js#L101-L103

Reply:
Done 😉.